modules/release-notes/pages/index.adoc

```diff
@@ -1,5 +1,9 @@
 = Insights Release Notes
 
+== 4.2
+* xref:insights:widgets:list-and-map.adoc[Support LIST and MAP query parameter types] - This allow users to create widgets in Insights that use GSQL queries with list or map parameters.
```

Comment:
```suggestion
* xref:4.2@insights:widgets:list-and-map.adoc[Support LIST and MAP query parameter types] - This allow users to create widgets in Insights that use GSQL queries with list or map parameters.
```
@MapleTang233 Added version specified to xref.  
Special case: On Release Notes, we know the reference should be to a specific version of documentation, so the links should explicitly include the version number. (There are other rules. Making xref do what you actually want with respect to versions is not as obvious as you might think.)